craft_providers/lxd/lxc.py

```diff
@@ -793,10 +793,17 @@ def project_create(self, *, project: str, remote: str = "local") -> None:
         try:
             self._run_lxc(command, capture_output=True, project=project)
         except subprocess.CalledProcessError as error:
-            raise LXDError(
-                brief=f"Failed to create project {project!r}.",
-                details=errors.details_from_called_process_error(error),
-            ) from error
+            if project in self.project_list(remote=remote):
```

Comment:
Can you add a comment along the lines of this?
`handle the race condition where two processes check and create a project at the same time`

Reply:
Sure